content/200-orm/500-reference/325-prisma-config-reference.mdx

```diff
@@ -122,46 +122,29 @@ export default {
 ```
 
 ### `schema`
-- Type: `object`
-- Required: No
-- Default: `{ kind: 'single', filePath: './prisma/schema.prisma' }`
-
-Configures how Prisma ORM locates and loads your schema file(s). See sub-options below for details.
-
-#### `schema.filePath`
-- Type: `string`
-- Required: Only when `schema.kind` is `'single'`
-- Default: `'./prisma/schema.prisma'`
-
-Specifies the path to your Prisma schema file when using a single-file setup.
 
-#### `schema.folderPath`
 - Type: `string`
-- Required: Only when `schema.kind` is `'multi'`
+- Required: No
 - Default: none
```

Comment:
❓ Shall we note here that `./primsa/schema.prisma` and `./schema.prisma` are our default search locations for schema files though? 🤔

Reply:
ah good call, I added this:

```md
- Type: `string`
- Required: No
- Default: `./prisma/schema.prisma` and `./schema.prisma`
```